src/pyff/builtins.py

```diff
@@ -438,15 +466,20 @@ def load(req, *opts):
         elif os.path.exists(url):
             if os.path.isdir(url):
                 log.debug("directory %s verify %s as %s via %s" % (url, params['verify'], params['as'], params['via']))
-                req.md.load_dir(url, url=params['as'], validate=opts['validate'], post=post)
+                req.md.load_dir(url, url=params['as'], validate=opts['validate'], post=post, fail_on_error=opts['fail_on_error'], filter_invalid=opts['filter_invalid'])
```

Comment:
I'm starting to wonder if we should refactor the argument passing to use named arguments instead. Opinions?